sdk/communication/Azure.Communication.CallAutomation/tests/SessionRecordedEvents/CallDialogAutomatedLiveTests/ConsecutiveDialogsTest.json

```diff
@@ -0,0 +1,199 @@
FTktjbUZYVVdsUGFVcHNWRlJTUWxaR1VuTmpNR1J1VFVkR1JWbHVTa3BrUlRWTVkxWk5NbHBxWXpGT1YzTXhWSHByZDFwcVRtcFdNRlphVDFob01WWXhUbEpKYmpFNVRFTktjRnBJUVdsUGFVcHZaRWhTZDJONmIzWk1NMDR3WTNrMU0yRlhOV3RpTTJSNlRHMDFiR1JET1dwYVIwMHhXVmRXYkZsVE1IaE9WMDB4VEZSU2ExbHFXWFJaYWtFelQxTXhiVmt5Um10YVJFa3hUVVJXYTFsNlNYWkphWGRwWWpKc2EwbHFiMmxOYlUwelQxZE9hVTU2WjNSTk1rNXFXVk13TUUweVZtbE1WMHBzV1dwSmRGbHFRVFZPYlZacFQxUkpNRTV0UlRSSmFYZHBZMjFuYVU5cFNYZE1hMFpUVTFWRk1tTlVaRWRsYlU1V1ZtNVNjazF1Wkd4YWJtd3dUVWQ0UTFwSVpHNWlNSEJaWTJ4T1prMXRkRmRSYlRsSVdqQnNlRkV5TVRaU1J6aDNWVEJHUWxGVE5HbE1RMHA2WkZkSmFVOXBTWGxaZW1NMVdUSkpNMDlETUhwWk1rNW9URlJSZWxwWFNYUlpiVlpwVFdreGFVMUVhekphVjBrMVRXcFJNbGxVWjJsTVEwb3dZVmRSYVU5cFNtcGFSMDB4V1ZkV2JGbFRNSGhPVjAweFRGUlNhMWxxV1hSWmFrRXpUMU14YlZreVJtdGFSRWt4VFVSV2ExbDZTV2xNUTBveFpFZHJhVTlwU1RCbGJUVXpVV3BHVDFsV1VrWmtWV2hDWWtaTk0wNXViR2hYUlVaQ1NXbDNhV1J0Vm5sSmFtOXBUVk0wZDBsdU1DNXllRFZpU1hKc2EyOVRiVmRqUVdWdlFWZHZSbTFJZFRkd2JuRkhWbmsyWTJ0QlpscDRlVEZCYmpWblZXbEpaWEZHTm5aemNWWnVZVkJ1V1hSbVkyTkdhM1J4V0dac1MxaGpiR2xsZFhGRmRHWlNSMUY1ZEhVMFVsUmFWbmMwVVUxNFdsa3RRVGRDT1Y5M2JEaGlWR1J6YkhCSFVtWTFhMVZUYzNCbVdIVm9OazExTlV4MVdrbHBjakJ5TlhKMWJYaFBjV1ZxZGs1MlZYcHRTVXBCYWw5dVgxRjRSVFZzZFhkaFVFRjZOVGcxU2xGb05HSkJTalpvVnpGMlpsQTVSbVJhZDBOTVYwNDBjVFJRZGpGUVRrTnpiMFIwTVVGR01HOXpiRjlYWlhsbVkyRlpZVVZQY0d4bVRGbFNNVUpFTjB0YU4xVlFNV2hmUm1wbmFXeHFRWFZIVG01bmVqaHhRMHRSUkhOSmVtMVRNazEzZGxreFIxUlBPVzVTYlZOb1FWWkZkRjlNWmt4b1ZVSkhka1Y2V21SMU5HdFNhbmswTW1kR1NYWjNSVVprT1hCSVVtOUJOVFoxUkU5c1h6ZHhMV2Q1TjNWelQzY2lMQ0owY3lJNk1UWTRPVGc0TWpRM055d2lZaUk2SWpJM2NuSllZWGhOU25KZlpIZ3pNMmd6V1hkeU1HNW1aRGxMVjNkSVIxWndlbWxITXpjNGNHUjBlVzhpTENKdWIyNWpaU0k2SWpnd05UTTNaamcwWXpSbE1qUmpaRGRpT0RFME9HUXhPV1F3WTJZMU5tVmlJaXdpWTI1bUlqcDdJbXAzYXlJNmV5SnJhV1FpT2lKMFRtUXpZblprVFVKc1dXaG9helV5YVdSMFEycGFWMk5PZW1oVk0yZHlaRlJWUjNCUllXVXRiV2hSSWl3aVpTSTZJa0ZSUVVJaUxDSnJkSGtpT2lKU1UwRWlMQ0p1SWpvaWMwWmtUbTAwVTBSa1IyNXhNbDluVVMxVVNYaG1ielJhV0hJelZYZEJTSHB1VjBOSkxWVlNVbFpDU3kxcWExUTNYMlJvVVRSTU5YbHNVelZTU214cmFuTk1TalppTVcxWFMwdFllSGsyYVdkTlltMXdlSGRXVUY5eFlrYzJjSEZXZVdsSlIydHpVbEpPWTBnMWJFUnJkRVoxVUhWbVN5MVlUbWxtVFRsaFdFcG5PVTVFWmt0U1NVTkxjVlpaT0U5RFZrNXZNRGxHY3pKU01GQkpjVXRFYjIweWRVczFRVEJxVWxWdlowaFdhblpuV0VWcFRscDZWSGRhUlhkSGVXTkdjMnBFVVdKd2NYZDZRMDVyVTJRMlRsbEtNRFZ2V25CbGNESkZVVEZEY2pOdFExRXhlV2RvZG5rMVNsUlBPSHB3ZGpKSE0zaEZlSHB5Y2pocE0xbGlkMnBQVjI5NE4zcFJSM1ExTFVvM1J6Qk9ZMkZrZFdkM2ExcHlWbG80TFRCeVlsRmZiRGxSYUdKamRWVjVYMEp4V1dwUE1VYzRWV294ZDBWaFlUUm9Ta051VUVsaGMwVkpiVVV6Ym01TWVEQnFWRGhUZUUxUkluMTlmUS5hbWNhSFQ4NEpFXzdzb3dDS2FEQ1AtWktzdFM3anQ5OUJDRkV0YmlEaU13RzFTNTdYc0RMQk5BUXpqWTZPXzRSc1dNY1VBenNBUU1mM3dSZ0hta0tGVFNuNHhuTlJqRjlPTnRZRUxBRGdVdFNrN29xaTE1bDczV0xNMy1GUU90Z21SWk90SU9OVXJ4ZUJfamdKRG04d25fWGx1anZHREtsTkJiTnM2ZHVMU1UyMmotbGpqT2h3WWdhWEZzSUVBdV9xZ3h4c0taS2dVVnNhTFFIbVZRUnZJS19yVjNHSnhjSnZ3YXljZjBZSS0xaEwzWl9qeG8yRkM1OFZhbU41VDZKNjJQTVBTV1RmRHRIZUFSWW1BcDhUY0FXRmN3OHViNnNmek5Wc3R2cWpEaDdqU2lnaEFpMWU0UDV6OThxbkJmRVhmT0NSNW5ab09kSEQtSTFQdDZIOHcifQ.\u0022,\u0022correlationId\u0022:\u0022d13d1fd3-0537-4909-b3d3-8d92601109a0\u0022}",
```

Comment:
should 18662315126 be sanitized?

Reply:
No, it's fine as it is a number specifically used for testing.